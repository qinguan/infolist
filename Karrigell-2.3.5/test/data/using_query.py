print "<br>Spam is ",QUERY["spam"]
if QUERY.has_key("animal"):
    print "<br>Animal is ",str(QUERY["animal"])
