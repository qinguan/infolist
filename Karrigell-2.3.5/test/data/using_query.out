<br>Spam is  %(spam)s
<br>Animal is  %(animal)s
